package.json

```diff
@@ -41,8 +41,8 @@
     "test:unit:watch": "cross-env NODE_ENV=test babel-node test/watch.js unit"
   },
   "peerDependencies": {
-    "react": "^15.4.0",
-    "react-dom": "^15.4.0",
+    "react": "^15.5.0",
+    "react-dom": "^15.5.0",
```

Comment:
Ah you're right for the `peerDependencies`. I do think you'll want to keep the `dependencies` (though I think in this case they should be `devDependencies`, but I don't have any historical context here) to `15.5.0` for `react-dom` because this diff uses `react-dom/test-utils`, which was a recent migration.

Just want to confirm with you before I make the change.